build/steps-build-ios.yml

```diff
@@ -103,7 +103,7 @@ steps:
     condition: failed()
 
 - task: Xcode@5
-  condition: and(succeeded(), eq(variables['IsPullRequestBuild'], 'false'))
```

Comment:
validation to isPR was removed to always get the ipa, otherwise it'll fail on Security scan step because wont find the file to be scanned
@lesterbotello @Soap-141